### PR TITLE
[UI Tests] Added missing dashboard mocks

### PR DESCRIPTION
**Pull request by pachlava**
### Description
This PR does two things:
- **Refactoring**: Updates an accessibility identifier for dashboard domains card content view from `domains-card-contentview` to `dashboard-domains-card-contentview` (I added the identifier in #20562)
- **Mocks**: Adds/updates mocks for requests that are used in dashboard

| Before | After |
|-------|-------|
| Screenshot 2023-04-21 at 22 06 22 | Screenshot 2023-04-21 at 22 07 38 |

### To test
- CI is green

### Regression Notes
I'm not filling the regression notes since the PR adds a mocks for UI tests, it does not change the app behaviour in any way.

### CODE REVIEW

File: API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/dashboard/domains.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+              "aftermarket_auction": false,
+              "aftermarket_auction_start": null,
+              "aftermarket_auction_end": null,
+              "nominet_pending_contact_verification_request": false,
+              "nominet_domain_suspended": false
+            }
+          ]
+      }
+    }
+  }
```
Comment (jostnes) [inline API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/dashboard/domains.json]: super nit: looks like it's missing the end of file line

**Review by jostnes — APPROVED** [review]
the mock and accessibility identifier changes look good and CI is green. one tiny nitpicky comment about the missing new line after the end of file but other than that it's good to :shipit:

File: API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/dashboard/domains.json (hunk 1 above)
Comment (pachlava) [reply API-Mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/dashboard/domains.json]: Thanks, @jostnes, I'll add it now by suggesting myself a change 😆 

```suggestion
  }

```

**Comment by tiagomar** [comment]
Thanks for adding those, @pachlava! 🙇‍♂️

**Pull request merged**